.github/workflows/ci.yml

```diff
@@ -292,3 +292,36 @@ jobs:
                  -F "force=true" \
                  "$url/api/v4/plugins"
           done
+
+  docker-mcp-server-build-push:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: ci/setup-qemu
+        uses: docker/setup-qemu-action@v3
+
+      - name: ci/setup-buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: ci/docker-login
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: ci/extract-version
+        id: version
+        run: echo "version=${GITHUB_REF#refs/tags/v}" >> $GITHUB_OUTPUT
+
```

Comment:
What do you think about adding a step to scan the Dockerfile & images on pull-request and before pushing to the repository. With that we can catch issues earlier in CI and before those images are published.